.github/workflows/go.yml

```diff
@@ -0,0 +1,40 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
```

Comment:
This CI is used for build each commit. Since we don't use dep, please remove `dep` installation. And we use `go mod (vgo)` instaed, so dependencies should be get using `go mod tidy` instead.

Reply:
It seems to be`go mod download`?

Reply:
Yes, `go mod download` for newly build environment.